sql/moz-fx-data-shared-prod/pocket/events/view.sql

```diff
@@ -0,0 +1,7 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.pocket.events`
+AS
+SELECT
+  * EXCEPT (submission_date)
```

Comment:
I don't understand why the time partitioning field is being excluded from the view. i see that submission_timestamp also exists, so you might want only one of the two to be available, but then i would expect submission_timestamp to be the time partitioning field.